scripts/release/util/smoke_test.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+# This is currently used by `test_package.sh`.
+# It is copied into a docker image at build time and then invoked at run time.
+
+GREEN_FG=$(tput setaf 2 2>/dev/null)
+RED_FG=$(tput setaf 1 2>/dev/null)
+END_FG_COLOR=$(tput sgr0 2>/dev/null)
+
+while [ "$1" != "" ]; do
+    case "$1" in
+        -b)
+            shift
+            BRANCH="$1"
+            ;;
+        -c)
+            shift
+            CHANNEL="$1"
+            ;;
+        -h)
+            shift
+            HASH="$1"
+            ;;
+        -r)
+            shift
+            RELEASE="$1"
+            ;;
+        *)
+            echo "Unknown option" "$1"
+            exit 1
+            ;;
+    esac
+    shift
+done
```

Comment:
could you please add verification that all the above parameters have been provided ( either via export or explicitly provided as command line arguments ? )

Reply:
Will do.